### DISCUSSION TITLE
[Mobile] Quick add transaction

### DISCUSSION THREAD

**Discussion by sapphirebound** [actualbudget/releases#152] [Feature ideas]
Currently the flow of recording transaction involves:
1. Opening the app
2. Going to Accounts
3. Pick account transaction was made
4. Create a transaction

Would be nice if there's a Material-like "+" button on bottom right of the screen to add a transaction. Perhaps in the future we can consider making an app shrotcut for this too?

Side note: I know I can probably achieve this with Tasker. Going to explore this and update.

**Comment by Deguello** [comment]
You can long press the Account piggie to bring up the "Add Transaction" window.

Edit: This should be made more discoverable somehow or split into its own button, imo.

**Comment by MikeBishop** [comment]
See #49.  (But also, if you use this pathway right now, you'll hit #47.)
